libcloud/compute/base.py

```diff
@@ -258,6 +258,34 @@ def destroy(self):
         """
         return self.driver.destroy_node(self)
 
+    def start(self, sync=True):
```

Comment:
Those are extension methods so please prefix them with ``ex_``.

Reply:
@Kami I was trying to underdtand the method naming idea but couldn't find out why some methods have prefix `ex_` but others don't have. `start` and `stop` look cleaner than `ex_start` and `ex_stop`. Could you clarify what kind of extension is meant here? I suppose all cloud providers should have ability to start and stop nodes

Reply:
I see, thanks for clarification.
Please let me know if you still want to change method names or it's ok as it now